hack/verify-staging-imports.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+
+kube::golang::setup_env
+
+
+for dep in $(find staging/src/k8s.io/* -maxdepth 0 -type d); do
+	set +e
+	dep=$(basename ${dep})
+	go list -f {{.Deps}} ./vendor/k8s.io/${dep}/... | sed -e 's/ /\n/g' - | grep k8s.io/kubernetes | grep -v vendor | LC_ALL=C sort -u 
```

Comment:
The appended `sort` will always return 0, won't it? Put a `| grep -e "."` behind to check for non-emptiness.